technic/init.lua

```diff
@@ -4,6 +4,14 @@ technic = rawget(_G, "technic") or {}
 technic.plus = true
 technic.version = 1.2
 
+technic.sounds = {}
+if minetest.get_modpath("default") then
+   technic.sounds = default
+end
+if minetest.get_modpath("mcl_sounds") then
+   technic.sounds = mcl_sounds
+end
```

Comment:
I think it would still be better to keep using default as a fallback (even if it is not coming from a mod named default):
```suggestion
if minetest.get_modpath("mcl_sounds") then
   technic.sounds = mcl_sounds
else
   technic.sounds = default
end
```